project/Build.scala

```diff
@@ -106,7 +106,7 @@ object Build extends sbt.Build {
     }
   }
 
-  val commonSettings = Seq(
+  lazy val commonSettings = inThisBuild(baseSettings) ++ Seq(
```

Comment:
It could also stay named "settings" or alternatively be named "buildSettings" (or "thisBuildSettings"?). Let me know if you prefer that.